web/src/features/fba/components/map/FBAMap.tsx

```diff
@@ -292,6 +292,9 @@ const FBAMap = (props: FBAMapProps) => {
     }
 
     setMap(mapObject)
+    return () => {
```

Comment:
What problem does this solve?

Reply:
For some reason with React 18, the 'double' rendering that happens when running a dev build led to two `FBAMaps` being rendered in ASA. This cleanup function effectively unmounts/removes the map from its parent div so we don't have two maps showing up.